Source/JavaScriptCore/runtime/OptionsList.h

```diff
@@ -579,7 +579,7 @@ bool hasCapacityToUseLargeGigacage();
     v(Bool, useWasmIPIntEpilogueOSR, true, Normal, "Allow IPInt to tier up during function epilogues"_s) \
     v(Bool, wasmIPIntTiersUpToBBQ, true, Normal, "Allow IPInt to tier up to BBQ"_s) \
     v(Bool, wasmIPIntTiersUpToOMG, true, Normal, "Allow IPInt to tier up to OMG"_s) \
-    v(Bool, useInterpretedJSEntryWrappers, false, Normal, "Allow some JS->Wasm wrappers to be replaced by jit-less versions."_s) \
+    v(Bool, useInterpretedJSEntryWrappers, true, Normal, "Allow some JS->Wasm wrappers to be replaced by jit-less versions."_s) \
```

Comment:
Let's not enable this until we fully support all cases in this operation. (It is fine to enable it for testing, but not enabling it right now when landing).

Reply:
Will do